### PR TITLE
button delete created

### PR DESCRIPTION
**Pull request by danielnottingham**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -10,7 +11,16 @@ def create
...
   end
 
+  def destroy
+    @user.destroy
+    redirect_to '/users'
+  end
+
   private
+    def set_user
+    @user = User.find(params[:id])
```
Comment (edimossilva) [inline app/controllers/users_controller.rb:21]: rubocop?

**Review by edimossilva — CHANGES_REQUESTED** [review]
1) Deploy not working.
2) test coverage print

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+require: rubocop-rails
```
Comment (edimossilva) [inline .rubocop.yml:1]: empty lane at the end of the files

Comment (danielnottingham) [reply .rubocop.yml]: ok

File: Gemfile.lock [hunk 3]
```diff
@@ -247,6 +272,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  aws-sdk-s3
```
Comment (edimossilva) [inline Gemfile.lock:275]: are you using this?

File: Profile [hunk 4]
```diff
@@ -0,0 +1 @@
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```
Comment (edimossilva) [inline Profile:1]: are you using this?

File: config/environments/production.rb [hunk 5]
```diff
@@ -1,5 +1,6 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  config.active_storage.service = :amazon
```
Comment (edimossilva) [inline config/environments/production.rb:3]: you are not using this, please remove

**Review by edimossilva — CHANGES_REQUESTED** [review]

File: config/environments/production.rb [hunk 6]
```diff
@@ -36,7 +37,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  #config.active_storage.service = :local
```
Comment (edimossilva) [inline config/environments/production.rb:40]: revert

File: config/puma.rb [hunk 7]
```diff
@@ -36,3 +36,19 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
```
Comment (edimossilva) [inline config/puma.rb:39]: are you using this?

File: Profile (hunk 4 above)
Comment (danielnottingham) [reply Profile:1]: because it was a config suggestion. 
create a profile file and send this command

File: config/puma.rb (hunk 7 above)
Comment (danielnottingham) [reply config/puma.rb:39]: because it was a config suggestion.
create a profile file and send this command

File: Profile (hunk 4 above)
Comment (edimossilva) [reply Profile:1]: what change if we remove or add it?

Comment (danielnottingham) [reply Profile:1]: the server already started on the puma, but as was suggested I wanted to try

File: Gemfile.lock (hunk 3 above)
Comment (edimossilva) [reply Gemfile.lock:275]: @danielnottingham this?

**Review by edimossilva — APPROVED** [review]

**Pull request merged**